### PR TITLE
Rename GS and add new Cloud GS (addresses #2124)

### PR DESCRIPTION
**Pull request by noramullen1**

### CODE REVIEW

File: content/influxdb/cloud/get-started.md [hunk 1]
```diff
@@ -1,299 +1,76 @@
 ---
 title: Get started with InfluxDB Cloud
-description: >
-  InfluxDB Cloud is a fully managed and hosted version of InfluxDB v2.0, the time series
-  platform purpose-built to collect, store, process and visualize metrics and events.
+description: Get started
 menu:
-  influxdb_cloud:
+  influxdb_2_0:
```
Comment (kelseiv) [inline content/influxdb/cloud/get-started.md:5]:
```suggestion
  influxdb_cloud:
```

File: content/influxdb/cloud/get-started.md [hunk 2]
```diff
@@ -1,299 +1,76 @@
...
-  - /influxdb/v2.0/cloud/get-started
+influxdb/v2.0/tags: [get-started, install]
 ---
 
-InfluxDB Cloud is a fully managed and hosted version of InfluxDB v2.0, the time series
-platform purpose-built to collect, store, process and visualize metrics and events.
+After you've signed up for and installed InfluxData, you're ready to start writing, querying, and visualizing data.
 
-_See [Differences between InfluxDB Cloud and InfluxDB OSS](#differences-between-influxdb-cloud-and-influxdb-oss)._
+Get started working with demo data in InfluxDB Cloud using notebooks using this guide.
```
Comment (kelseiv) [inline content/influxdb/cloud/get-started.md:13]:
```suggestion
To get started working in Cloud, we recommend using demo data and notebooks. Complete the following steps:
1. [Add demo data](add-demo-data)
2. ...
```

File: content/influxdb/cloud/get-started.md [hunk 3]
```diff
@@ -1,299 +1,77 @@
...
+2.  Click **{{< icon "plus" >}} Add Demo Data**, and then select the **Website Monitoring** bucket.
+3.  The Demo Data bucket appears in your list of buckets.
 
-    To subscribe to an InfluxDB Cloud **Free Plan** through InfluxData,
-    go to [InfluxDB Cloud](https://cloud2.influxdata.com/).
+## Select metrics
 
-    - To use social sign-on, click **Google** or **Microsoft**. Note that social sign-on does not support email aliases.
-    - Sign up with email by entering your name, email address, and password, then click **Create Account**.
+Now that we've added the demo data bucket, we're ready to create our notebook. The first notebook cell selects the bucket and metrics we'll work with.
```
Comment (sanderson) [inline content/influxdb/cloud/get-started.md:30]:
```suggestion
Now that we've added the demo data bucket, we're ready to create our notebook.
```

File: content/influxdb/cloud/get-started.md [hunk 4]
```diff
@@ -1,299 +1,77 @@
...
+3. Enter a name for your notebook in the **Name this notebook** field.
+4. By default, a **Metric Selector** and **Visualization** cell appear (see [Cell types](#cell-types) below for details.)
+5. In the **Metric Selector** cell, click the **Select a bucket** dropdown and choose **Website Monitoring Bucket**.
 
-    - **AWS**  
-      Sign in to AWS, navigate to the [InfluxDB Cloud product on AWS Marketplace](https://aws.amazon.com/marketplace/pp/B08234JZPS/?href=_ptnr_web_docs_gettingstarted),
-      and follow the prompts to subscribe. After you click **Set Up Your Account**,
-      enter your credentials, and then click **Start Now**.
-      All usage charges will be paid through the subscribed AWS account.
+{{< img-hd src="/img/influxdb/gs_select_bucket.png" alt="Select bucket" />}}
```
Comment (sanderson) [inline content/influxdb/cloud/get-started.md:40]: - I'm guessing you need to indent this to properly nest this in the `li` and not break the `ol` numbering.
- Update the image file name to use the [image naming conventions](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#image-naming-conventions):

    ```
    /img/influxdb/cloud-gs-select-bucket.png
    ```

File: content/influxdb/cloud/get-started.md [hunk 5]
```diff
@@ -1,299 +1,77 @@
...
 
-    - **Microsoft**  
-      Sign in to Microsoft Azure, navigate to the [InfluxDB Cloud product on Azure Marketplace](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/influxdata.influxdb-cloud?ocid=iflxdbcloud_influxdata_docs_gettingstarted),
-      and follow the prompts to subscribe. After you click **Set Up Your Account**,
-      enter your credentials, and then click **Start Now**.
-      All usage charges will be paid through the subscribed Microsoft account.
+A list of measurements, fields, and tags for the Website Monitoring bucket appear.
```
Comment (sanderson) [inline content/influxdb/cloud/get-started.md:42]:
```suggestion
    A list of measurements, fields, and tags for the Website Monitoring bucket appear.
```

File: content/influxdb/cloud/get-started.md [hunk 6]
```diff
@@ -1,299 +1,77 @@
...
-      and follow the prompts to subscribe. After you click **Set Up Your Account**,
-      enter your credentials, and then click **Start Now**.
-      All usage charges will be paid through the subscribed GCP account.
+6. For this example, click the `response_time` field.
+7. Click **Preview** to preview the raw response time data in a table.
 
-      {{%note%}}
-Currently, we do **not support** using an existing InfluxDB Cloud account to sign up for an InfluxDB Cloud plan through AWS, Microsoft, or GCP Marketplaces.
-      {{%/note%}}
+{{< img-hd src="/img/influxdb/gs_select_bucket.png" alt="Preview table" />}}
```
Comment (sanderson) [inline content/influxdb/cloud/get-started.md:47]: Update the image file name to use the image naming conventions:

```
/img/influxdb/cloud-gs-select-bucket.png
```

File: content/influxdb/cloud/get-started.md [hunk 7]
```diff
@@ -1,299 +1,77 @@
...
-   - To keep the free plan, click **Keep**. {{< cloud-name >}} opens with a default organization and bucket (both created from your email address). _To update organization and bucket names, see [Update an organization](/influxdb/cloud/organizations/update-org/)
-    and [Update a bucket](/influxdb/cloud/organizations/buckets/update-bucket/#update-a-buckets-name-in-the-influxdb-ui)._
-   - To upgrade to an Annual plan, click **Contact Sales**, enter your information, and then click **Send**. Our team will contact you as soon as possible.
+1. By default, a visualization cell appears below your initial metric selector cell showing the mean `response_time` values on a graph:
+2. Use the function dropdown menu to view different functions to apply to your data. For this response time data, the mean function is a good fit. <add link>
```
Comment (sanderson) [inline content/influxdb/cloud/get-started.md:55]:
```suggestion
2. Use the function dropdown menu to view different functions to apply to your data. For this response time data, the mean function is a good fit.
```

File: content/influxdb/cloud/get-started.md [hunk 8]
```diff
@@ -1,299 +1,77 @@
...
+3. Use the visualization type dropdown to view different visualization types available.
+For a complete list of options, see [Visualization types](/influxdb/cloud/visualize-data/visualization-types/).
+4. Click the gear icon ({{< icon "gear" >}}) to edit specific settings for the visualization type, such as colors, orientation, and labels.
 
-## (Optional) Download, install, and use the influx CLI
+## Process data
 
-To use the `influx` CLI to manage and interact with your InfluxDB Cloud instance, complete the following steps:
+A common processing task is downsampling data to view it at different levels of granularity and reduce overall data disk usage over time. The following steps use a Flux script to downsample the website monitoring sample data to every 12 hours.
```
Comment (sanderson) [inline content/influxdb/cloud/get-started.md:62]:
```suggestion
InfluxDB lets you process queried data in a number of ways. A common processing task is downsampling data to view it at different levels of granularity and reduce overall data disk usage over time. The following steps use a Flux script to downsample the website monitoring sample data to every 12 hours.
```

File: content/influxdb/cloud/get-started.md [hunk 9]
```diff
@@ -1,299 +1,77 @@
...
 
-{{< tabs-wrapper >}}
-{{% tabs %}}
-[macOS](#)
-[Linux](#)
-{{% /tabs %}}
+1. Click **+** to add a cell, then select **Transform > Downsample**.
```
Comment (sanderson) [inline content/influxdb/cloud/get-started.md:64]:
```suggestion
1. Click **{{< icon "plus" >}}** to add a cell, then select **Transform > Downsample**.
```

File: content/influxdb/cloud/get-started.md [hunk 10]
```diff
@@ -1,299 +1,77 @@
...
-  Flux can be used with other data sources as well, letting you work with data where it resides.
-- **Unified API**: Everything in InfluxDB (ingest, query, storage, and visualization)
-  is now accessible using a unified [InfluxDB v2 API](/influxdb/cloud/reference/api/) that
-  enables seamless movement between open source and cloud.
-- **Integrated visualization and dashboards**: Based on the pioneering Chronograf project,
-  the new user interface (InfluxDB UI) offers quick and effortless onboarding,
-  richer user experiences, and significantly quicker results.
-- **Usage-based pricing**: The [Usage-based Plan](/influxdb/cloud/account-management/pricing-plans/#usage-based-plan)
-  offers more flexibility and ensures that you only pay for what you use.
+1. Click **+** to add a cell, then select an **Output to Bucket** cell.
```
Comment (sanderson) [inline content/influxdb/cloud/get-started.md:73]:
```suggestion
1. Click **{{< icon "plus" >}}** to add a cell, then select an **Output to Bucket** cell.
```

File: content/influxdb/cloud/sign-up.md [hunk 11]
```diff
@@ -0,0 +1,297 @@
...
+   - To upgrade to an Annual plan, click **Contact Sales**, enter your information, and then click **Send**. Our team will contact you as soon as possible.
+
+## (Optional) Download, install, and use the influx CLI
+
+To use the `influx` CLI to manage and interact with your InfluxDB Cloud instance, complete the following steps:
+
+{{< tabs-wrapper >}}
+{{% tabs %}}
+[macOS](#)
+[Linux](#)
```
Comment (sanderson) [inline content/influxdb/cloud/sign-up.md:89]: We need to add Windows instructions here. The content for this change is on master.

File: content/influxdb/cloud/sign-up.md [hunk 12]
```diff
@@ -0,0 +1,297 @@
...
+{{% /tabs %}}
+
+<!-------------------------------- BEGIN macOS -------------------------------->
+{{% tab-content %}}
+
+#### Step 1: Download influx CLI for macOS
+
+Click the following button to download and install `influx` CLI for macOS.
+
+<a class="btn download" href="https://dl.influxdata.com/influxdb/releases/influxdb2-client-2.0.4-darwin-amd64.tar.gz" download>influx CLI (macOS)</a>
```
Comment (sanderson) [inline content/influxdb/cloud/sign-up.md:99]:
```suggestion
<a class="btn download" href="https://dl.influxdata.com/influxdb/releases/influxdb2-client-2.0.5-darwin-amd64.tar.gz" download>influx CLI (macOS)</a>
```

File: content/influxdb/cloud/sign-up.md [hunk 13]
```diff
@@ -0,0 +1,297 @@
...
+
+**Note:** The commands below are examples. Adjust the file names, paths, and utilities to your own needs.
+
+To unpackage the downloaded archive, **double click the archive file in Finder**
+or run the following command in a macOS command prompt application such
+**Terminal** or **[iTerm2](https://www.iterm2.com/)**:
+
+```sh
+# Unpackage contents to the current working directory
+tar zxvf ~/Downloads/influxdb2-client-2.0.4-darwin-amd64.tar.gz
```
Comment (sanderson) [inline content/influxdb/cloud/sign-up.md:111]:
```suggestion
tar zxvf ~/Downloads/influxdb2-client-2.0.5-darwin-amd64.tar.gz
```

File: content/influxdb/cloud/sign-up.md [hunk 14]
```diff
@@ -0,0 +1,297 @@
...
+#### Step 3: (Optional) Place the binary in your $PATH
+
+If you choose, you can place `influx` in your `$PATH` or you can
+prefix the executable with `./` to run in place. If the binary is on your $PATH, you can run `influx` from any directory. Otherwise, you must specify the location of the CLI (for example, `./influx`or `path/to/influx`).
+
+**Note:** If you have the 1.x binary on your $PATH, moving the 2.0 binary to your $PATH will overwrite the 1.x binary because they have the same name.
+
+```sh
+# Copy the influx binary to your $PATH
+sudo cp influxdb2-client-2.0.4-darwin-amd64/influx /usr/local/bin/
```
Comment (sanderson) [inline content/influxdb/cloud/sign-up.md:123]:
```suggestion
sudo cp influxdb2-client-2.0.5-darwin-amd64/influx /usr/local/bin/
```

File: content/influxdb/cloud/sign-up.md [hunk 15]
```diff
@@ -0,0 +1,297 @@
...
+
+<!-------------------------------- BEGIN Linux -------------------------------->
+{{% tab-content %}}
+
+#### Step 1: Download influx CLI for Linux
+
+Click one of the following buttons to download and install the `influx` CLI appropriate for your chipset.
+
+<a class="btn download" href="https://dl.influxdata.com/influxdb/releases/influxdb2-client-2.0.4-linux-amd64.tar.gz" download >influx CLI (amd64)</a>
+<a class="btn download" href="https://dl.influxdata.com/influxdb/releases/influxdb2-client-2.0.4-linux-arm64.tar.gz" download >influx CLI (arm)</a>
```
Comment (sanderson) [inline content/influxdb/cloud/sign-up.md:168]:
```suggestion
<a class="btn download" href="https://dl.influxdata.com/influxdb/releases/influxdb2-client-2.0.5-linux-amd64.tar.gz" download >influx CLI (amd64)</a>
<a class="btn download" href="https://dl.influxdata.com/influxdb/releases/influxdb2-client-2.0.5-linux-arm64.tar.gz" download >influx CLI (arm)</a>
```

File: content/influxdb/cloud/sign-up.md [hunk 16]
```diff
@@ -0,0 +1,297 @@
...
+
+#### Step 2: Unpackage the influx binary
+
+**Note:** The commands below are examples. Adjust the file names, paths, and utilities to your own needs.
+
+```sh
+# Unpackage contents to the current working directory
+tar xvfz influxdb-client-2.0.4-linux-amd64.tar.gz
```
Comment (sanderson) [inline content/influxdb/cloud/sign-up.md:176]:
```suggestion
tar xvfz influxdb-client-2.0.5-linux-amd64.tar.gz
```

File: content/influxdb/cloud/sign-up.md [hunk 17]
```diff
@@ -0,0 +1,297 @@
...
+#### Step 3: (Optional) Place the binary in your $PATH
+
+If you choose, you can place `influx` in your `$PATH` or you can
+prefix the executable with `./` to run in place. If the binary is on your $PATH, you can run `influx` from any directory. Otherwise, you must specify the location of the CLI (for example, `./influx`or `path/to/influx`).
+
+**Note:** If you have the 1.x binary on your $PATH, moving the 2.0 binary to your $PATH will overwrite the 1.x binary because they have the same name.
+
+```sh
+# Copy the influx and influxd binary to your $PATH
+sudo cp influxdb-client-2.0.4-linux-amd64/influx /usr/local/bin/
```
Comment (sanderson) [inline content/influxdb/cloud/sign-up.md:188]:
```suggestion
sudo cp influxdb-client-2.0.5-linux-amd64/influx /usr/local/bin/
```

File: content/influxdb/cloud/sign-up.md [hunk 18]
```diff
@@ -0,0 +1,297 @@
...
+  It is the primary language for working with data within InfluxDB Cloud.
+  Flux can be used with other data sources as well, letting you work with data where it resides.
+- **Unified API**: Everything in InfluxDB (ingest, query, storage, and visualization)
+  is now accessible using a unified [InfluxDB v2 API](/influxdb/cloud/reference/api/) that
+  enables seamless movement between open source and cloud.
+- **Integrated visualization and dashboards**: Based on the pioneering Chronograf project,
+  the new user interface (InfluxDB UI) offers quick and effortless onboarding,
+  richer user experiences, and significantly quicker results.
+- **Usage-based pricing**: The [Usage-based Plan](/influxdb/cloud/account-management/pricing-plans/#usage-based-plan)
+  offers more flexibility and ensures that you only pay for what you use.
```
Comment (sanderson) [inline content/influxdb/cloud/sign-up.md:297]: I think we could probably remove this content and put it into its own doc. This has changed and will continue to change. We need to be better about updating it.

File: content/influxdb/cloud/sign-up.md [hunk 19]
```diff
@@ -0,0 +1,297 @@
...
+{{< /tabs-wrapper >}}
+
+## Sign in
+
+Sign in to [InfluxDB Cloud](https://cloud2.influxdata.com) using your email address and password.
+
+<a class="btn" href="https://cloud2.influxdata.com">Sign in to InfluxDB Cloud now</a>
+
+## Start working with your time series data
+
```
Comment (sanderson) [inline content/influxdb/cloud/sign-up.md:228]: Add a link to the new get started doc?

File: content/influxdb/v2.0/install.md [hunk 20]
```diff
@@ -1,12 +1,13 @@
...
   influxdb_2_0:
-    name: Get started
+    name: Install 
 weight: 2
 influxdb/v2.0/tags: [get-started, install]
 aliases:
+  - /influxdb/v2.0/get-started/
   - /influxdb/v2.0/introduction/get-started/
   - /influxdb/v2.0/introduction/getting-started/
 ---
```
Comment (sanderson) [inline content/influxdb/v2.0/install.md:13]: Let's not make this change as part of this PR. At this point, I'm not comfortable simply renaming the OSS get-started doc. We need to have both an Install and a Get Started doc in OSS. Remove this change from the PR and we can do it in another.

**Review by sanderson — CHANGES_REQUESTED** [review]
Summary of feedback/suggestions:

- Let's keep this PR cloud focused. Revert all the changes to the OSS docs until we can address them fully.
- Update the image names to match the naming convention.
- Add Windows install instructions (from master)
- Update versions to 2.0.5
- Other minor suggestions

File: content/influxdb/cloud/sign-up.md (hunk 18 above)
Comment (kelseiv) [reply content/influxdb/cloud/sign-up.md:297]: Moving forward, we should prob just mention "new" features in Release Notes. As for this content, we could drop free plan/usage-based pricing given we discuss earlier, and mention Flux support, unified API, and dashboards in intro paragraph of this doc...or leave that content to the get started doc, either or.

**Review by kelseiv — CHANGES_REQUESTED** [review]

File: content/influxdb/cloud/get-started.md [hunk 21]
```diff
@@ -1,299 +1,77 @@
...
 ---
 
-InfluxDB Cloud is a fully managed and hosted version of InfluxDB v2.0, the time series
-platform purpose-built to collect, store, process and visualize metrics and events.
+After you've signed up for and installed InfluxData, you're ready to start writing, querying, and visualizing data.
 
-_See [Differences between InfluxDB Cloud and InfluxDB OSS](#differences-between-influxdb-cloud-and-influxdb-oss)._
+To get started working in Cloud, we recommend using demo data and notebooks. Complete the following steps:
+1. [Add demo data](add-demo-data)
+2. ...
```
[the remark this answers is not in the document]
Comment (noramullen1) [reply content/influxdb/cloud/get-started.md:16]: Weird, I must've not committed all my changes originally.

File: content/influxdb/cloud/sign-up.md [hunk 22]
```diff
@@ -0,0 +1,234 @@
+---
+title: Sign up for InfluxDB Cloud
+description: >
+  InfluxDB Cloud is a fully managed and hosted version of InfluxDB v2.0, the time series
```
Comment (kelseiv) [inline content/influxdb/cloud/sign-up.md:4]:
```suggestion
  InfluxDB Cloud is a fully managed and hosted version of InfluxDB 2.0, the time series
```

File: content/influxdb/cloud/sign-up.md [hunk 23]
```diff
@@ -0,0 +1,234 @@
...
+  platform purpose-built to collect, store, process and visualize metrics and events.
+menu:
+  influxdb_cloud:
+    name: Sign up
+weight: 1
+influxdb/cloud/tags: [get-started, install]
+---
+
+InfluxDB Cloud is a fully managed and hosted version of InfluxDB v2.0, the time series
```
Comment (kelseiv) [inline content/influxdb/cloud/sign-up.md:13]:
```suggestion
InfluxDB Cloud is a fully managed and hosted version of InfluxDB 2.0, the time series
```

**Review by sanderson — CHANGES_REQUESTED** [review]
- Don't rename `content/influxdb/v2.0/get-started` to `content/influxdb/v2.0/install` in this PR. Save that change for another PR.
- Remove the `package.json`. I assume this got added trying to fix some npm issues.
- Add Windows instructions to Cloud getting started. You can get an idea of the instructions from [here](https://github.com/influxdata/docs-v2/blob/master/content/influxdb/v2.0/get-started.md#experimental-windows-support). They only need the "InfluxDB Cloud CLI" from the downloads page, but I'd add a download button directly in this doc.
- Update filname of `static/img/gs_preview_raw.png` to `static/img/cloud-gs-preview-raw.png`

File: content/influxdb/cloud/sign-up.md [hunk 24]
```diff
@@ -0,0 +1,262 @@
...
+<!--------------------------------- END Linux --------------------------------->
+
+<!-------------------------------- BEGIN Windows -------------------------------->
+{{% tab-content %}}
+
+#### Step 1: Download influx CLI for Windows
+
+Click the following button to download and install `influx` CLI for Windows.
+
+<a class="btn download" href="https://dl.influxdata.com/influxdb/releases/influxdb2-2.0.6-windows-amd64.tar.gz" download>influx CLI (macOS)</a>
```
Comment (sanderson) [inline content/influxdb/cloud/sign-up.md:233]:
```suggestion
<a class="btn download" href="https://dl.influxdata.com/influxdb/releases/influxdb2-2.0.6-windows-amd64.tar.gz" download>influx CLI (Windows)</a>
```

**Review by sanderson — CHANGES_REQUESTED** [review]
Looks like when you changed the name of `install.md` back to `get-started.md`, it replaced all the content with an old version of the `get-started` page. You need to copy all the get-started markdown from master and update the content of `get-started.md`.

It did this because your name change was in your git history, so when git replays the changes in order, it sees a file rename (with old content), updates to the get-started on master, merge those changes into the branch, then essentially a file replace (install -> get-started) but the content of the install doc was outdated.

**Pull request merged**